silver/tests/unit/test_transaction.py

```diff
@@ -0,0 +1,45 @@
+from django.test import TestCase
+from django_fsm import TransitionNotAllowed
+from silver.models import Transaction, Payment
+from silver.tests.factories import TransactionFactory, PaymentFactory
+
+TStates = Transaction.State
+PStates = Payment.Status
```

Comment:
You could do transaction_object.States and payment_object.states, but I guess it doesn't matter that much.